view/frontend/web/js/action/dintero-payment-service.js

```diff
@@ -28,6 +28,7 @@ define(
                     storage.post(serviceUrl, JSON.stringify(payload), true, 'application/json')
                         .then(function(session, status, request) {
                             _this.currentRequest = request;
+                            $('#dintero-embedded-checkout-container').html('');
```

Comment:
Can you explain the problem a bit?

Reply:
@andersem , the problem is that `then` block is executed only when Ajax receives a response from a server and if the server responds a bit slowly it is still possible to send the second Ajax request and get 2 embedded iframes within same `container`. That is why we clean the HTML right before initializing Dintero Checkout iframe.